README.md

```diff
@@ -98,13 +103,13 @@ Create a `local.json` file inside the config folder:
         "google": {
             "client_id": "<google client id>",
             "client_secret": "<google client secret>",
-            "callback_url": "http://lvh.me:3001/google/callback"
+            "callback_url": "http://localhost:3001/google/callback"
         }
     }
 }
 ```
 
-Credentials can be found in the [Google App]() (TBD).
+Credentials can be found in the [Google App](https://console.cloud.google.com/apis/credentials?project=fizca-310305).
```

Comment:
I'd leave the project query params out. Google will resolve to the last project seen and the person can change to their project. I don't think we should put the google developer info here just in case.

Reply:
Sure, so just https://console.cloud.google.com/apis then?

Reply:
oh, slash creds...updating